nornir/core/deserializer/configuration.py

```diff
@@ -65,30 +65,32 @@ def deserialize(cls, **kwargs) -> configuration.InventoryConfig:
 
 
 class LoggingConfig(BaseSettings):
-    level: str = Schema(default="debug", description="Logging level")
-    file: str = Schema(default="nornir.log", descritpion="Logging file")
+    enabled: bool = Schema(
+        default=True, description="Whether to configure logging or not"
+    )
+    level: str = Schema(default="DEBUG", description="Logging level")
+    file_path: str = Schema(default="nornir.log", description="Logging file")
     format: str = Schema(
-        default="%(asctime)s - %(name)12s - %(levelname)8s - %(funcName)10s() - %(message)s",
+        default="[%(asctime)s] %(levelname)-8s {%(name)s:%(lineno)d} %(message)s",
```

Comment:
breaking change, might break people's logging system

Reply:
How is it breaking? It's just the format of logs.